doc/changelog.rst

```diff
@@ -31,6 +32,9 @@ Description
 
 Detailed Notes
 
+- Users can now specify node features to a job through
+  SrunSettings.set_node_feature. The method accepts a string
```

Comment:
Should we put SrunSettings.set_node_feature in ticks as `SrunSettings.set_node_feature` or whatever is needed by .rst syntax?

Reply:
yes! good catch